acceptancetests/utility.py

```diff
@@ -294,14 +294,17 @@ def add_basic_testing_arguments(
     parser.add_argument('--bootstrap-host',
                         help='The host to use for bootstrap.')
     parser.add_argument('--machine', help='A machine to add or when used with '
-                        'KVM based MaaS, a KVM image to start.',
+                                          'KVM based MaaS, a KVM image to start.',
                         action='append', default=[])
     parser.add_argument('--keep-env', action='store_true',
                         help='Keep the Juju environment after the test'
-                        ' completes.')
+                             ' completes.')
     parser.add_argument('--logging-config',
                         help="Override logging configuration for a deployment.",
                         default="<root>=INFO;unit=INFO")
+    parser.add_argument('--cloud-city', help="Directory of cloud city. It is not used during integrationtest runs. "
```

Comment:
Rename - the name will remain nameless!

```suggestion
    parser.add_argument('--juju-home', help="Directory of cloud city. It is not used during integration test runs. "
```